packages/nuxeo-platform-explorer-package/src/main/resources/install/templates/explorer-deploy/nuxeo.prod

```diff
@@ -0,0 +1,153 @@
+
+
+## s3binaries confs mapping
+
+# mandatory
+nuxeo.s3storage.bucket=${env:NUXEO_S3STORAGE_BUCKET:}
+
+# usually taken from system environement
+nuxeo.s3storage.awsid=${env:NUXEO_S3STORAGE_AWSID:}
+nuxeo.s3storage.awssecret=${env:NUXEO_S3STORAGE_AWSSECRET:}
+nuxeo.s3storage.awstoken=${env:NUXEO_S3STORAGE_AWSTOKEN:}
+nuxeo.s3storage.region=${env:NUXEO_S3STORAGE_REGION:}
+
+# optional
+nuxeo.s3storage.cacheminage=${env:NUXEO_S3STORAGE_CACHEMINAGE:60}
+nuxeo.s3storage.cachesize=${env:NUXEO_S3STORAGE_CACHESIZE:100MB}
+nuxeo.s3storage.cachecount=${env:NUXEO_S3STORAGE_CACHECOUNT:10000}
+nuxeo.s3storage.connection.max=${env:NUXEO_S3STORAGE_CONNECTION_MAX:50}
+nuxeo.s3storage.connection.retry=${env:NUXEO_S3STORAGE_CONNECTION_RETRY:3}
+nuxeo.s3storage.connection.timeout=${env:NUXEO_S3STORAGE_CONNECTION_TIMEOUT:50000}
+nuxeo.s3storage.socket.timeout=${env:NUXEO_S3STORAGE_SOCKET_TIMEOUT:50000}
+nuxeo.s3storage.endpoint=${env:NUXEO_S3STORAGE_ENDPOINT:}
+nuxeo.s3storage.pathstyleaccess=${env:NUXEO_S3STORAGE_PATHSTYLEACCESS:false}
+nuxeo.s3storage.accelerateMode=${env:NUXEO_S3STORAGE_ACCELERATEMODE:false}
+nuxeo.s3storage.bucket_prefix=${env:NUXEO_S3STORAGE_BUCKET_PREFIX:}
+nuxeo.s3storage.crypt.keystore.file=${env:NUXEO_S3STORAGE_CRYPT_KEYSTORE_FILE:}
+nuxeo.s3storage.crypt.keystore.password=${env:NUXEO_S3STORAGE_CRYPT_KEYSTORE_PASSWORD:}
+nuxeo.s3storage.crypt.key.alias=${env:NUXEO_S3STORAGE_CRYPT_KEY_ALIAS:}
+nuxeo.s3storage.crypt.key.password=${env:NUXEO_S3STORAGE_CRYPT_KEY_PASSWORD:}
+nuxeo.s3storage.crypt.serverside=${env:NUXEO_S3STORAGE_CRYPT_SERVERSIDE:false}
+
+nuxeo.s3storage.useDirectUpload=${env:NUXEO_S3STORAGE_USEDIRECTUPLOAD:false}
+
+# mandatory
+nuxeo.s3storage.transient.bucket=${env:NUXEO_S3STORAGE_TRANSIENT_BUCKET:}
+nuxeo.s3storage.transient.roleArn=${env:NUXEO_S3STORAGE_TRANSIENT_ROLEARN:}
+nuxeo.s3storage.transient.targetMaxSizeMB=${env:NUXEO_S3STORAGE_TRANSIENT_TARGETMAXSIZEMB:100}
+nuxeo.s3storage.transient.ttl=${env:NUXEO_S3STORAGE_TRANSIENT_TTL:4320}
+nuxeo.s3storage.transient.ttl2=${env:NUXEO_S3STORAGE_TRANSIENT_TTL2:10}
+
+# usually taken from system environement
+nuxeo.s3storage.transient.awsid=${env:NUXEO_S3STORAGE_TRANSIENT_AWSID:}
+nuxeo.s3storage.transient.awssecret=${env:NUXEO_S3STORAGE_TRANSIENT_AWSSECRET:}
+nuxeo.s3storage.transient.awstoken=${env:NUXEO_S3STORAGE_TRANSIENT_AWSTOKEN:}
+nuxeo.s3storage.transient.region=${env:NUXEO_S3STORAGE_TRANSIENT_REGION:}
+
+# optional
+nuxeo.s3storage.transient.endpoint=${env:NUXEO_S3STORAGE_TRANSIENT_ENDPOINT:}
+nuxeo.s3storage.transient.expiration=${env:NUXEO_S3STORAGE_TRANSIENT_EXPIRATION:}
+nuxeo.s3storage.transient.pathstyleaccess=${env:NUXEO_S3STORAGE_TRANSIENT_PATHSTYLEACCESS:false}
+nuxeo.s3storage.transient.accelerateMode=${env:NUXEO_S3STORAGE_TRANSIENT_ACCELERATEMODE:false}
+nuxeo.s3storage.transient.bucket_prefix=${env:NUXEO_S3STORAGE_TRANSIENT_BUCKET_PREFIX:}
+nuxeo.s3storage.transient.crypt.serverside=${env:NUXEO_S3STORAGE_TRANSIENT_CRYPT_SERVERSIDE:false}
```

Comment:
Let's use the default, non S3 based transient store.

Reply:
removed